example/src/components/StandardTextInput.tsx

```diff
@@ -0,0 +1,89 @@
+import { AdditionalColors, BrandColors, TextColors } from '@colors';
+import { TextInputTextStyle } from '@components/Typo';
+import React, { useState } from 'react';
+import { StyleSheet, TextInput } from 'react-native';
+
+const StandardTextInputStyles = StyleSheet.create({
```

Comment:
can't we just name it `TextInput`? Adding nothing makes it standard :p

Reply:
Renamed styles and props.

Reply:
@skyman503 What about a component?

Reply:
I though it's better not to override default names to avoid confusion later on. However, I tried doing that but it makes es-lint angry because of: `'TextInput' is already defined. eslint[@typescript-eslint/no-redeclare]` in the `StandardTextInput.tsx`. Idk whether ignoring that is a good idea. WDYT @graszka22 ?

Reply:
FIxed.